### PR TITLE
Rewrote header functions as inline, add toString to all objects

### PR DESCRIPTION
**Pull request by Kyew**

### CODE REVIEW

File: src/DspAdc.h [hunk 1]
```diff
@@ -44,4 +44,12 @@ class DspAdc : public DspObject {
...
 
+inline const char *DspAdc::getObjectLabel() {
+  return "adc~";
+}
+
+inline std::string DspAdc::toString() {
+  return DspAdc::getObjectLabel();
+}
+
 #endif // _DSP_ADC_H_
```
Comment (mhroth) [inline src/DspAdc.h]: newline at EOF

File: src/DspClip.h [hunk 2]
```diff
@@ -1,47 +1,51 @@
-/*
```
Comment (mhroth) [inline src/DspClip.h:1]: What happened here? Replace tabs with spaces?

File: src/DspCosine.h [hunk 3]
```diff
@@ -33,9 +33,9 @@ class DspCosine : public DspObject {
     DspCosine(PdMessage *initMessage, PdGraph *graph);
     ~DspCosine();
 
-    static const char *getObjectLabel() { return "cos~"; }
-    string toString() { return string(getObjectLabel()); }
-
+    static const char *getObjectLabel();
+    std::string toString();
+
```
Comment (mhroth) [inline src/DspCosine.h]: Remove trailing space.

File: src/DspInlet.h [hunk 4]
```diff
@@ -39,19 +39,39 @@ class DspInlet : public DspObject {
...
-    ObjectType getObjectType() { return DSP_INLET; }
+    static const char *getObjectLabel();
+    std::string toString();
+    ObjectType getObjectType();
   
     list<DspObject *> getProcessOrder();
     list<DspObject *> getProcessOrderFromInlet();
   
     // [inlet~] does nothing with audio
-    bool doesProcessAudio() { return false; }
```
Comment (mhroth) [inline src/DspInlet.h:50]: This function doesn't really have to be `inline`, but this will be ok.

File: src/DspInlet.h [hunk 5]
```diff
@@ -39,19 +39,39 @@ class DspInlet : public DspObject {
...
+    bool doesProcessAudio();
   
     void setDspBufferAtInlet(float *buffer, unsigned int inletIndex);
-    bool canSetBufferAtOutlet(unsigned int outletIndex) { return false; }
```
Comment (mhroth) [inline src/DspInlet.h:53]: Doesn't really have to be inline.

**Comment by mhroth** [comment]
Just a few small changes and this PR will be good.

File: src/DspClip.h (hunk 2 above)
Comment (Kyew) [inline src/DspClip.h:1]: I'm not really sure anymore, but it might be yes.

File: src/DspInlet.h (hunk 5 above)
Comment (Kyew) [inline src/DspInlet.h:53]: Do you want me to correct this ?

Comment (mhroth) [inline src/DspInlet.h:53]: I don't have a strong opinion. What is your opinion?

**Pull request merged**